manifests/init.pp

```diff
@@ -159,6 +162,7 @@
   Apt::Proxy $proxy                               = $apt::params::proxy,
   Hash $sources                                   = $apt::params::sources,
   Hash $keys                                      = $apt::params::keys,
+  Hash $keyrings                                  = $apt::params::keyrings,
```

Comment:
please directly assign the data here:
```suggestion
  Hash $keyrings                                  = {},
```

And please replace the `Hash` datatype with something that's a bit stricter.